staging/src/k8s.io/sample-controller/README.md

```diff
@@ -112,6 +112,7 @@ Enable this feature using the `CustomResourceSubresources` feature gate on the [
 ```sh
 --feature-gates=CustomResourceSubresources=true
 ```
+It is beta in v1.11 and enabled by default.
```

Comment:
better to first explain the state of the newer version?

---------------

Custom Resources support `/status` and `/scale` subresources as an
[beta feature](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#subresources) in v1.11 and is enabled by default.
The feature is alpha in in v1.10 and to enable it you need to set the `CustomResourceSubresources` feature gate on the [kube-apiserver](https://kubernetes.io/docs/admin/kube-apiserver):

```sh
--feature-gates=CustomResourceSubresources=true
```

Reply:
good idea. I'll fix it soon